lib/package-map.js

```diff
@@ -223,6 +223,8 @@ const packageMap = [
           'SkyAppResources',
           'skyLibResources',
           'SkyLibResources',
+          'SkyAppResourcesService',
+          'SkyAppResourcesTestingService',
```

Comment:
`SkyAppResourcesTestingService` is actually located in `@skyux/i18n/testing` (separate module). 
See: https://github.com/blackbaud/skyux-i18n/tree/master/src/app/public/testing

Reply:
Oh good catch.  Is it in a module?

Reply:
Is this right @Blackbaud-SteveBrush?  https://github.com/blackbaud/skyux-sdk-builder-plugin-migrate/pull/32/files#diff-46027fd2460dbf051c4893b0938a1c16R233